apps/www/app/templates/client.tsx

```
@@ -37,6 +37,9 @@ import { useEffect, useMemo } from "react";
 import { useForm } from "react-hook-form";
 import { type Framework, type Language, templates } from "./data";
 
+// New reusable sorting function
+const sortAlphabetically = (a: [string, number], b: [string, number]) => a[0].localeCompare(b[0]);
```

Biome

[error] This variable is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.
Unsafe fix: If this is intentional, prepend sortAlphabetically with an underscore.


(lint/correctness/noUnusedVariables)